mkl2017-xeon-dnn/mkl/pom.xml

```diff
@@ -73,6 +73,7 @@
                         <linkerStartOption>-mkl=parallel</linkerStartOption>
                         <linkerStartOption>-static-intel</linkerStartOption>
                         <linkerStartOption>-qopenmp</linkerStartOption>
+                        <linkerStartOption>-no-intel-extensions</linkerStartOption>
```

Comment:
fix: remove warnning  #10237: -lcilkrts linked in dynamically, static library not available